capsul/engine/settings.py

```diff
@@ -273,10 +273,8 @@ def export_config_dict(self, environment=None):
                     env_conf = {}
                     for module in modules:
                         mod_conf = {}
-                        for config in session.configs(module, env):
-                            id = "%s-%s" % (config._id, env)
-                            doc = data[session.collection_name(module)][id]
-                            items = dict(doc._items())
```

Comment:
Once `get()` is called, the value of the document is returned (very likely as a dict in this case). There is no `_items()` method in ` Storage` API.

Reply:
Maybe the test `if "config_id" in doc:`  could be appropriate here ?

Reply:
`dict()` is useless here.
Personally, I would have directly set `doc = data[session.collection_name(module)][id].get()` and replace `items` by ` doc`. But this is no important.